guide-gen/guide-configs/concepts/cloud.json

```
@@ -0,0 +1,47 @@
+{
+  "file_name": "index.md",
+  "id": "index",
+  "file_dir": "cloud",
+  "title": "Temporal Cloud documentation",
+  "sidebar_label": "Temporal Cloud",
+  "description": "Teporal Cloud documentation explanation and usage information.",
```

```suggestion
  "description": "Temporal Cloud documentation, including explanations and usage.",
```
Or something like that.